style/responsiveness.css

```diff
@@ -19,4 +15,93 @@
         flex-direction: column;
         gap: 3;
     }
+    .header__item {
+        padding: 2rem 23px;
+    }
+    .header__navigation {
+        display: none;
+    }
+
+    .section-header {
+        text-align: center;
+        font-size: 40px;
+        margin-top: 2rem;
+    }
+    .header__button-container {
+        flex-direction: column;
+    }
+    .section-header {
```

Comment:
`section-header` is twice repeated in this code , on line 25 and 33 , the line 33 overwrite the margin setted on line 25, try to fixe this